deploy-jenkins.sh

```diff
@@ -14,8 +14,16 @@ do
 done
 ${DM_CREDENTIALS_REPO}/sops-wrapper -d ${DM_CREDENTIALS_REPO}/aws-keys/ci.pem.enc > $PRIVATE_KEY_FILE
 
+# Override inventory if running on localhost
+if [ -z ${LOCALHOST+x} ]; 
```

Comment:
I think this might have a typo?

Reply:
@lfdebrux I took this syntax from https://stackoverflow.com/a/13864829/238230

Reply:
Ah, interesting, I knew about `${parameter:+word}` but not `${parameter+word}`!